packages/ava/README.md

```diff
@@ -22,35 +33,40 @@ $ npm install @antv/ava --save
 $ yarn add @antv/ava
 ```
 
-## Usage
+The following is a practical example of how the four main modules of [@antv/ava](https://www.npmjs.com/package/@antv/ava) can be used:
 
 ```ts
-import { Advisor, ckb, DataFrame, getInsights } from '@antv/ava';
+import { DataFrame, getInsights, ckb, Advisor } from '@antv/ava';
 
-// Chart Advisor and Linter
+// Input data
 const data = [
   { price: 38, type: 'A' },
   { price: 52, type: 'B' },
   { price: 61, type: 'C' },
   { price: 145, type: 'D' },
   { price: 49, type: 'E' },
 ];
-const chartAdvisor = new Advisor();
-const results = chartAdvisor.advise({ data });
-
-// Chart Knowledge Base
-const myCkb = ckb();
 
-// Data Processing and Analysis
+// 1. Data Processing Module
 const df = new DataFrame(data);
 const dataInfo = df.info();
 
-// Data Insight
+// 2. Automatic Insights Module
```

Comment:
L41 capitalizes the first letter of the first word only, L50 and L54 capitalize the first letter of all words, plz unify them.
https://github.com/antvis/AVA/pull/704/files#diff-884bd56149db70e7927ab4546a4ef2af42fa374ef9818a5e6f17414b4c8bacf2R41